paper.md

```diff
@@ -76,4 +76,10 @@ Figures can be included like this: ![Fidgit deposited in figshare.](figshare_art
 
 # Main Body
 
+## Story of the project
+
+We are a group of graduate students who connected through the [UofT Coders](http://uoftcoders.github.io/) club at the University of Toronto. We created and ran a fourth-year course in the Department of Ecology and Evolutionary Biology titled “Theoretical Ecology and Reproducible Quantitative Methods in R.” We modelled the structure and portions of the course content on “Reproducible Quantitative Methods” a course created by Dr. Christie Bahlai [ ], modifying the lesson content to include more theoretical ecology topics but maintaining the focus on reproducibility and computational skills. Our teaching team was six graduate students from several different fields (Physiology, Biomedical Engineering, Physics, Psychology, and Nutritional Science); we divided course topics among each instructor to develop and deliver individual lessons and assignments.
```

Comment:
club -> group

Reply:
`[“Reproducible Quantitative Methods”](https://cbahlai.github.io/rqm-template/)`

Reply:
It might be nice to say *why* we started the course, but that might be too much words already.

Reply:
Yeah, good point - it could be a short thing, like we saw a need for more quantitative skills for biology students.

Reply:
I can add this within the next hour. @lwjohnst86 may I repurpose from the rostools application?

Reply:
@lwjohnst86 Nvm, I am not, but almost done

Reply:
@mbonsma I added a PR on your branch with a new paragraph and Luke's comments. Maybe it's too long now... =/ Is there are word limit?

Reply:
The whole thing should be about 1000 words, but we can do the trimming at the end.